### ISSUE TITLE
[20220410] Weekly AI ArXiv 만담 - 48회차 (DALLE-2 특집)

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#48]
### News
- 컨퍼런스 소식
  - ICLR 2022 다음주 부터 (4.25 ~ 29) 시작입니다: https://iclr.cc/
    - 네이버 클로바의 ICLR 스케쥴 (12개): https://naver-career.gitbook.io/en/teams/clova-cic/events/naver-clova-iclr-2022
    - ICLR Social: ML in Korea 도 학회기간 진해예정입니다! (곧 공유 예정)
- Jigsaw: MS의 디버깅 AI - Copilot에 적용 (Python only) --> 이준형님께 토스
  - https://www.zdnet.com/article/sorry-developers-microsofts-new-tool-fixes-the-bugs-in-software-code-written-by-ai/
  - [한국어 기사](https://news.naver.com/main/read.naver?mode=LSD&mid=shm&sid1=105&oid=092&aid=0002254059)
### ArXiv
- [Masked Siamese Networks for Label-Efficient Learning](https://arxiv.org/abs/2204.07141)
  - Few-shot에 강한 새로운 SSL (from Meta AI)
  - MAE + DINO의 좀더 일반화된 버전: Masked patch, EMA 와 clustering 기반 softmax prediction matching, stop gradient 등
  - 실제 주요 실험도 label 1% 사용 혹은 클래스당 이미지 극소로 사용하는 것이 주력
  - Linear probing과 FT둘다 경쟁력있어보임
![image](https://user-images.githubusercontent.com/11782739/163705009-bc56d6f6-9f0b-49a9-aa38-0986995b7851.png)
![image](https://user-images.githubusercontent.com/11782739/163705021-c0833247-dfa5-4058-9a2c-43ebb03d8727.png)

- 주목할 만한 논문
  - [DeiT III: Revenge of the ViT](https://arxiv.org/abs/2204.07118): ViT의 역습! --> 다음 주에 이진원님이
  - [GPT-NeoX-20B: An Open-Source Autoregressive Language Model](https://arxiv.org/abs/2204.06745)
    - Eleuther AI의 NeoX-20B 논문이 나왔어요!
  - [KOBEST: Korean Balanced Evaluation of Significant Tasks](https://arxiv.org/abs/2204.04541) 
    - 5개의 조금더 리즈닝이 필요한 한국어 NLU 벤치마크셋 + SKT의 초거대언어모델 39B
    - KorQuAD 나 AI Hub를 소개하지 않은 것도 아쉬움. 
  - [The Moral Integrity Corpus: A Benchmark for Ethical Dialogue Systems](https://arxiv.org/abs/2204.03021)
    - 윤리적인 내용을 고려한 대화 벤치마크 데이터
  - [Using Interactive Feedback to Improve the Accuracy and Explainability of Question Answering Systems Post-Deployment](https://arxiv.org/abs/2204.03025)
    - QA시스템을 deploy한 후에 Human in the loop으로 시스템을 계속 개선할 수 있는 방법

**Comment by ghlee0304** [comment]
- Arxiv (Speech and Music) (오늘은 DALL-E 2 특집이므로 짤막하게 언급만하고 넘어갈게요)
    - 출처 / 소속 / 키워드  
    - [Analysis and transformations of intensity in singing voice](https://arxiv.org/pdf/2204.04006.pdf)
    - [Karaoker: Alignment-free singing voice synthesis with speech training data](https://arxiv.org/pdf/2204.04127.pdf)
    - [Hierarchical and Multi-Scale Variational Autoencoder for Diverse and Natural Non-Autoregressive Text-to-Speech](https://arxiv.org/pdf/2204.04004.pdf)

**Comment by nick-jhlee** [comment]
- Amii presents AI Week (05/24~05/27)
   - https://www.ai-week.ca/
   - Amii: [Alberta Machine Intelligence Institute](https://www.amii.ca/)
     - Affiliated researchers: [Richard Sutton](http://incompleteideas.net/), [Csaba Szepesvári](https://sites.ualberta.ca/~szepesva/index.html), [Mijung Park](https://privacy-preserving-machine-learning.github.io/)...etc
   - (이미 끝났지만) Talent Bursaries도 줬어요...! <캐나다에서 열리는 행사라, 만약 talent bursaries에 당첨이 되면, 북아메리카를 제외한 나머지 나라들의 신진 연구자들 및 대학원생들에겐 300만원을 줍니당>
   - Programs:
      - academic: Keynotes, AI Meetup, Public Coworking, Talent Mixer, VIP Talent Mixer, AI Seminar, Academic Symposium
      - social: Lunchbox Short, Coffee Catchup, Afternoons with Amii, Beers with Peers, Amii House Party, VIP Dinner, Amiiversary Party, AI Brunch

**Comment by epicure** [comment]
DALL-E 2에 관한 이야기 

- 미디어 아트 계의 반응
- 거의 모든 것에 관한 보간
- DALL·E 2 Preview - Risks and Limitations 행간 읽기

(계속)

**Comment by hollobit** [comment]
### DALL-2의 편향성 문제와 LGM의 문제 
https://www.vox.com/future-perfect/23023538/ai-dalle-2-openai-bias-gpt-3-incentives

>- OpenAI에서도 이미 알고 공지하고 있는 이슈 - https://github.com/openai/dalle-2-preview/blob/main/system-card.md?utm_source=Sailthru&utm_medium=email&utm_campaign=Future%20Perfect%204-12-22&utm_term=Future%20Perfect
>- "충분한 가드레일이 없으면 DALL·E 2와 같은 모델을 사용하여 광범위한 기만 및 기타 유해 콘텐츠를 생성할 수 있으며 사람들이 콘텐츠의 진위를 보다 일반적으로 인식하는 방식에 영향을 미칠 수 있습니다. DALL·E 2는 또한 훈련 데이터에서 다양한 편향을 상속하고 그 출력은 때때로 사회적 고정 관념을 강화합니다"
>- 충분하게 위험을 통제할 수 없는 사항이기에 선택한 방법 : 현재로서는 약 400명(OpenAI의 직원과 이사회 구성원, 엄선된 학자 및 창작자 혼합)만이 비상업적 목적으로만 DALL-E 2를 사용
>- AI 산업의 잘못된 인센티브 문제를 어떻게 해결해야 하나 ? 
>    * AI 안전 및 연구 회사인 Anthropic의 팀에서는  Large Generative Models에서의 이런 문제점을 지적 - https://arxiv.org/pdf/2202.07785.pdf
>    *  경제적, 과학적, 명성이라는 대규모 생성 모델을 개발하고 배포하는 세 가지 주요 동기, 반대로 모델을 확장하는 데 필요한 재정적 비용과 엔지니어링 인재, AI 안전 문제, 모델 배포의 표준 및 규범 부족이라는 세 가지 진입 장벽
>    * 대규모 모델이 확산되고 있고, 산업계/학계의 격차가 늘어나고 있고, 피해와 논쟁이 증가하고 있는 현실 
<img  src="https://user-images.githubusercontent.com/998803/163706406-d3797cc0-6812-43da-a539-f524528d9e8b.png">

>- Anthropic 연구팀 백서에서 제안하는 해결 방향
>    * 민간 부문과 학계 간의 컴퓨팅 비대칭 감소
>    * 모델 '레드 팀' 방법에 대한 지식 향상
>    * 새로운 거버넌스 구조 및 정부 개입 탐색 및 프로토타입
>    * 모델 평가에 사용할 수 있는 도구 개선
>    * 능력의 급격한 점프(급격한 능력 확장)에 대한 이해 향상  

>- OpenAI가 DALL-E 2에 대해 한 것처럼 모델의 위험을 문서화하는 사례들이 계속 늘어나야하며, 일반화시켜야 함

**Comment by veritas9872** [comment]
**Jigsaw: Large Language Models meet Program Synthesis**

논문 및 공식 블로그 링크 공유해드립니다.

Blog: https://www.microsoft.com/en-us/research/blog/jigsaw-fixes-bugs-in-machine-written-software/

Arxiv: https://arxiv.org/abs/2112.02969
